classes/core-image-pelux.bbclass

```diff
@@ -19,6 +19,12 @@ IMAGE_INSTALL_append = "\
     ${@bb.utils.contains("COMBINED_FEATURES", "bluetooth", "packagegroup-tools-bluetooth", "", d)} \
 "
 
+# Include udev-extraconf for automatic mounting of pluggable mass storage
+# devices (other parts are stripped out).
+IMAGE_INSTALL_append = "\
```

Comment:
We should stop adding IMAGE_INSTALL for every component and perhaps have a few generic sections.

Reply:
Hum, sure... but what do you suggest more concretely at this point? I can not see a way to group things together in a nice way at the moment. Perhaps just remove some comments and then just have 1 or 2... perhaps even 3... IMAGE_INSTALL? Most comments do not add anything... some things should not be here... should we have a "should be moved out of here" section?

Reply:
I do not have a clear suggestion atm. Let's discuss this separately then. You can proceed with the current change.

Reply:
I added an extra commit on top with a suggestion for how to reduce the current number of IMAGE_INSTALL_append.